src/components/Dashboard/Form/FieldWrapper/FieldWrapper.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../../../scss/_variables.scss';
+
+.default{
+  font-family: 'Poppins';
+  font-size: $font-xs;
+  line-height: $line-height-sm;
+  padding: 10px 0;
+  margin-right: 25px;
+}
+
+.small {
+  composes: default;
```

Comment:
Is this required? If we don't declare this -  if css/scss don't should set `default` automatically?